applications/StructuralMechanicsApplication/custom_response_functions/response_utilities/adjoint_strain_energy_response_function.cpp

```diff
@@ -100,16 +100,15 @@ namespace Kratos
         rResponseGradient.clear();
 
         const double numerical_limit = std::numeric_limits<double>::epsilon();
-        Vector acc_1 = ZeroVector(3);
-        Vector acc_2 = ZeroVector(3);
+        Vector acc = ZeroVector(3);
         if (rAdjointElem.GetProperties().Has( VOLUME_ACCELERATION ))
```

Comment:
We could create a single private function for this, because the code looks the same in both cases.

Reply:
good idea